docs/_docs/templates.md

```diff
@@ -4,8 +4,8 @@ permalink: /docs/templates/
 ---
 
 Jekyll uses the [Liquid](https://shopify.github.io/liquid/) templating language to
-process templates. All of the standard Liquid [tags](https://shopify.github.io/liquid/tags/) and
-[filters](https://shopify.github.io/liquid/filters/) are
+process templates. All of the standard Liquid [tags](https://help.shopify.com/themes/liquid/tags) and
+[filters](https://help.shopify.com/themes/liquid/filters) are
 supported. Jekyll even adds a few handy filters and tags of its own to make
```

Comment:
The links you changed to, actually documents the [Shopify variant of Liquid](http://shopify.github.io/liquid/basics/variations/) instead of [Standard Liquid](http://shopify.github.io/liquid/basics/introduction/)

Please update.